build/props.js

```diff
@@ -82,6 +84,32 @@ const figmatokensSYNC = { 'open-props': { ...figmatokens } }
 const FigmaTokensSync = fs.createWriteStream('../open-props.figma-tokens.sync.json')
 FigmaTokensSync.end(JSON.stringify(figmatokensSYNC, null, 2))
 
+if (!fs.existsSync('../dist'))
+  fs.mkdirSync('../dist')
+
+const JS = fs.createWriteStream('../dist/open-props.js')
+JS.end(`var OpenProps = ${JSON.stringify(toObject(), null, 2)}`)
+
+const ES = fs.createWriteStream('../dist/open-props.module.js')
+ES.end(`export default ${JSON.stringify(toObject(), null, 2)}`)
+
+const CJS = fs.createWriteStream('../dist/open-props.cjs')
+CJS.end(`module.exports = ${JSON.stringify(toObject(), null, 2)}`)
+
+// const UMD = fs.createWriteStream('../dist/open-props.umd.js')
+// UMD.end(`module.exports = ${JSON.stringify(toObject(), null, 2)}`)
+
+const TS = fs.createWriteStream('../dist/open-props.ts')
+TS.end(`export default ${JSON.stringify(toObject(), null, 2)}`)
+
+const TSD = fs.createWriteStream('../dist/open-props.d.ts')
+TSD.end(`declare const OpenProps: ${JSON.stringify(toTypes(), null, 2).replaceAll(',',';')};\nexport default OpenProps;`)
+
+preparedTypes().forEach(({filename, json}) => {
+  let bundle = fs.createWriteStream('../dist/props.'+filename+'.d.ts');
```

Comment:
this should probably be `src` not `dist` because that's where the js files can be found
```suggestion
  let bundle = fs.createWriteStream('../src/props.'+filename+'.d.ts');
```